### ISSUE TITLE
Trouble with Pycharm Data Viewer Integration [JDBC]: Column Names broken

### ISSUE THREAD

**Issue by tiansrc** [sassoftware/saspy#464]
# PyCharm Database Table Viewer shows only comments/labels or "<anonymous>" as Column Name

### Situation
We connected a SAS Dataservice with PyCharm (and Intellij, DataGrip) succesfully.

- All tables and column information in the database tool tab looks fine….
- But, if we open a table, the column name on all culomns is "<anonymous>"
- Only Columns with a given comment/label will show the comment text.
- Checking the DDL we found out, that each column without a comment has a plenty of blanks as a comment :-(

### Describe the solution you'd like
On empty Labels the blanks should be suppressed, there should be no Label (xyz as  '                                                ' )
(I know, this is not the JDBC team, but may you can double check this behaviour, know a solution or know the java colleagues...)

### Describe alternatives you've considered
A JDBC Option to ignore Labels and do sth like
```
public String getLabel() {
        return this.name;
    }
```
### Additional context

https://youtrack.jetbrains.com/issue/DBE-14103/Database-Table-Colum-Name-anonymous-instead-Colum-Name

![image](https://user-images.githubusercontent.com/56220492/170675464-cd03df03-e917-4bc9-8ce2-11be012050f6.png)

![image](https://user-images.githubusercontent.com/56220492/170675519-1adcd930-04ee-401a-b3ec-4bc089a3ccda.png)

```
904600.1.0.20181024190000_v940m6
java version "1.8.0_321"
Java(TM) SE Runtime Environment (build 1.8.0_321-b32)
Java HotSpot(TM) Client VM (build 25.321-b32, mixed mode)
```

**Comment by tomweber-sas** [comment]
Have you opened a SAS Tech Support track for this? That will be the best wat to get this addressed.

**Comment by tiansrc** [comment]
not yet, we are still on validating this behaviour, but i think we have to ...

**Comment by tomweber-sas** [comment]
Ok, and yes, I think that would be best. I don't personally know about those drivers, while TS will get to the right person who knows that area and can help.

**Comment by tomweber-sas** [comment]
Hey, I'm going to close this, as there's nothing I can do to help. It's not a SASPy thing, and I don't know enough to be helpful w/ the drivers you're trying to use. I'd help if I could, but I've got nothing. Let me know if there is anything I can help with.

Thanks,
Tom

**Comment by tiansrc** [comment]
Jetbrains have implemented a workaround. 
Fixed in 2022.2 (Checked on EAP)
Details: https://youtrack.jetbrains.com/issue/DBE-14103/

**Comment by tiansrc** [comment]
Release 2022.2 reviewed!
PyCharm now shows names of columns if they don't have column labels 😎
(Up to now this columns were showed as "anonymus")
![image](https://user-images.githubusercontent.com/56220492/186114012-df0944d2-146d-4307-9ba3-999f03b02a7b.png)
